CONTRIBUTING.md

```diff
@@ -55,9 +71,8 @@ executable will be in your current directory and can be run as shown in the
   the associated `.pb.{go,cc,h}` or `.js` files using `make generate`.
 
 - We advise to run `make generate` using our embedded Docker setup.
-  `build/builder.sh` is a wrapper script designed to make this convenient. You
-  can run `build/builder.sh make generate` from the repository root to get the
-  intended result.
+  You can run `build/builder.sh make generate` from the repository
```

Comment:
nit: since you removed the preceding sentence, I think this would read better as "You can also use the builder image via `build/builder.sh make generate`..."

Reply:
Done